decompiler/config/jak1_ntsc_black_label/label_types.jsonc

```diff
@@ -1446,6 +1446,376 @@
   "steam-cap": [
     ["L51", "attack-info"]
   ],
+  
+  "sidekick": [
+    ["L29", "vector"]
+  ],
+
+  "target": [
+    ["L720", "float", true],
+    ["L721", "float", true],
+    ["L722", "float", true],
+    ["L723", "float", true],
+    ["L724", "float", true],
+    ["L725", "float", true],
+    ["L726", "float", true],
+    ["L727", "float", true],
+    ["L728", "float", true],
+    ["L729", "float", true],
+    ["L730", "float", true],
+    ["L731", "float", true],
+    ["L732", "float", true],
+    ["L733", "float", true],
+    ["L734", "(pointer float)", 1],
+    ["L735", "(pointer float)", 1],
+    ["L736", "(pointer float)", 1],
+    ["L737", "(pointer float)", 1],
+    ["L738", "(pointer float)", 1],
```

Comment:
I think you can change all these `["L734", "(pointer float)", 1],` to `["L734", "float", true],` and it makes the code a bit cleaner.

Reply:
Has the underlying issue been fixed?  Without this workaround things fail to work properly.

Reply:
it should be fine?

Reply:
I'll have to do a pass and check them then, before they were broken for these gigantic files in certain situations.

Reply:
Nope, the issue still persists.
```asm
L482:
    lui v1, L734              ;; [525] (set! v1-176 L734) [] -> [v1: <label-765> ]
    ori v1, v1, L734
    addu v1, fp, v1
    lw v1, 0(v1)              ;; [526] (set! v1-175 (l.w v1-176)) [v1: <label-765> ] -> [v1: int ]
```

Reply:
I think the `lw` from float far labels doesn't work or something like that